tests/mbed/enc/CMakeLists.txt

```diff
@@ -91,6 +91,12 @@ function(add_mbed_selftest_enclave)
     ${CMAKE_BINARY_DIR}/3rdparty/mbedtls/mbedtls-wrap-prefix/src/mbedtls-wrap/include/)
   target_link_libraries(mbedtest_selftest oelibc)
 
+  # The following variables remain unitialized on Linux and throw warnings
```

Comment:
uninitialized*

I am curious why these throw warnings, can we link to an explanation?

Reply:
The function called here : https://github.com/Microsoft/openenclave/blob/83730e9ce20d9e436f86f8eae0190f692888d623/tests/mbed/enc/CMakeLists.txt#L100 is the one that is setting those variables.  In our case , running inside docker container , the function return is empty
https://github.com/Microsoft/openenclave/blob/master/cmake/maybe_build_using_clangw.cmake#L25-L32